Cargo.toml

```diff
@@ -99,16 +99,17 @@ workspace = true
 vello = { workspace = true }
 thiserror = "1.0.69"
 usvg = "0.44.0"
-image = { version = "0.25.5", default-features = false, features = [
-  "webp",
-  "png",
-  "jpeg",
-  "gif",
-] }
+image = { version = "0.25.5", default-features = false, features = [ "webp", "png", "jpeg", "gif"], optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
 wasm-bindgen-test = "0.3.49"
 
 [features]
+default = ["image", "image_format_png", "image_format_gif", "image_format_jpeg", "image_format_webp"]
```

Comment:
I don't know if `image` needs to be included here since it is transitively enabled. Up to you though.